packages/k8s/src/k8s/index.ts

```diff
@@ -137,6 +152,16 @@ export async function createJob(
     }
   ]
 
+  if (extension) {
+    if (extension.metadata) {
+      mergeObjectMeta(job, extension.metadata)
```

Comment:
leave `//` with explaining how a container step is wrapped into a k8s job and we want to apply the metadata extensions to both